kamon-logback-stackdriver/src/test/scala/kamon/stackdriver/StackdriverEncoderTest.scala

```diff
@@ -65,3 +73,10 @@ class StackdriverEncoderTest extends AnyFlatSpec with Matchers {
   }
 
 }
+
+object StackdriverEncoderTest {
+  case class TestStackdriverMarker(name: String) extends StackdriverMarker {
```

Comment:
Probably for testing purposes you should use BasicMarker form slf4j or something like that.

Reply:
It has private constructor, I would stay with this one, it is cleaner

Reply:
I mean there is no existing Marker out there that I could use, we would have to add some on classpath

Reply:
Well, you can create BasicMarker using MarkerFactory, check https://github.com/qos-ch/slf4j/blob/master/slf4j-api/src/test/java/org/slf4j/BasicMarkerTest.java

Reply:
Problem of testing this one is that it overrides StackdriverMarker, so it is treated differently than this one would.